src/buildCommon.js

```diff
@@ -330,11 +330,13 @@ const makeLineSpan = function(
     className: string,
     options: Options,
 ) {
-    // Fill the entire span instead of just a border. That way, the min-height
-    // value in katex.less will ensure that at least one screen pixel displays.
-    const line = stretchy.ruleSpan(className, options);
-    line.height = options.fontMetrics().defaultRuleThickness;
-    line.style.height = line.height + "em";
+    // Return a span with an SVG image of a horizontal line. The SVG path
+    // fills the middle third of the span. We want an extra tall span
```

Comment:
Should this be `fills the middle fifth of the span.`?

Reply:
Yep.